website/sidebars.json

```diff
@@ -6,49 +6,39 @@
       "web-app-boilerplate",
       "backend-boilerplate",
       "open-source-projects",
-      "developer-tools",
-      {
-        "type": "subcategory",
-        "label": "Visual Resources",
```

Comment:
We should keep a page called "Visual Resources" and link to the individual pages from there rather than from the menu

Reply:
I think @AngeloCG97  do that in this issue #72

Reply:
Correcto